changelog.txt

```diff
@@ -1 +1,8 @@
-
+feature - Implements anonymous user upgrade functionality with FirebaseUI.
+feature - Implements isPendingRedirect needed to tell whether there is a pending redirect opearation.
```

Comment:
opearation -> operation.
While you're at it, isPendingRedirect -> `isPendingRedirect`